tools/offline_log_viewer/storage.py

```diff
@@ -104,6 +105,36 @@ def __next__(self):
                       headers)
 
 
+class BatchType(Enum):
```

Comment:
This is good -- controller.py already has some of these magic numbers in, so maybe those could be replaced with references to this class at the same time as adding it?

Reply:
Done. I did some sanity testing on my local controller log and kvstore data and things seem ok.